frontend/src/lib/components/CommandPalette/CommandResults.tsx

```diff
@@ -10,7 +10,7 @@ interface CommandResultProps {
     result: CommandResultType
     handleSelection: (result: CommandResultType) => void
     focused?: boolean
-    setHoverResultIndex: Dispatch<SetStateAction<number | undefined | null>>
+    setHoverResultIndex: Dispatch<SetStateAction<number | undefined>>
```

Comment:
why `undefined` and not `null`? 

In my mind `null` is more like "explicitly not there" and undefined is "not there by default". Plus if you ever store an `undefined` value in redux/kea, it'll throw an error, so it's wise to be consistent.